docs/workspaces.md

```diff
@@ -0,0 +1,146 @@
+# Workspaces
+
+## Why workspaces?
+
+The JW OTT Web App is an open-source repository that showcases an OTT app implementing JWP services. The OTT Web App, as
+the name implies, originates as a web only repository. But much of the source-code can be re-used for many different
+platforms. Like; Capacitor, React Native, and other frameworks based on TypeScript.
+
+Using the previous codebase, it would be quite challenging to re-use the services because of the dependencies and
+browser usage. For example, the AccountController could redirect a user to a different page by using `window.location`.
+This will never work in a non-browser environment and will crash the app.
+
+This means that we need to:
```

Comment:
```suggestion
This means that we had to:
```

Reply:
I like it more in the current form instead of the past (relatable to my previous comment)